docs/DesignDocs/ExtendedEnclaveInitializationData.md

```diff
@@ -0,0 +1,154 @@
+Extended Enclave Initialization Data
+=====
+
+We present an extension of Intel SGX attestation in OpenEnclave that enables the attestation of enclave initialization parameters.
```

Comment:
Line 1 is not SGX specific, but line 4 is SGX specific.  Which is it?  Please correct one of the two lines.

Personally I think that it should NOT be SGX specific, and the design below is too SGX specific.

Reply:
I don't think the concept itself requires anything SGX-specific, so I agree that we should keep the design document platform independent. Currently, all we need is an SGX implementation though and I think that would be fine with the idea of a specific SGX attestation plugin.